### PR TITLE
Rename folder and update url for visual e2e

### PR DESCRIPTION
**Pull request by nadvolod**

### CODE REVIEW

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Can you remove all screener references in this PR, file names, config name and so on? Otherwise this one is a half PR

**Comment by nadvolod** [comment]
@wswebcreation if it's good, please merge

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This repository will hold multiple NodeJS based automation frameworks and exampl
 - [Protractor with Jasmine](./protractor)
 - [Puppeteer with Jest](./puppeteer)
 - [Screener Runner](./screener-runner)
-- [Screener E2E](./screener-e2e) (*in beta*)
+- [Screener E2E](visual-e2e) (*in beta*)
```
Comment (wswebcreation) [inline README.md:19]: There is still Screener here in the name

File: visual-e2e/examples/test/configs/wdio.screener.saucelabs.conf.js [hunk 2]
```diff
@@ -14,8 +12,8 @@ config.capabilities = [
         platformName: 'macOS 10.15',
         browserVersion: 'latest',
         'sauce:options': {
-            username: config.user,
-            accesskey: config.accessKey
+            username: process.env.SAUCE_USERNAME,
+            accesskey: process.env.SAUCE_ACCESS_KEY
         },
         'sauce:visual': {
             apiKey: process.env.SCREENER_API_KEY,
```
Comment (wswebcreation) [inline visual-e2e/examples/test/configs/wdio.screener.saucelabs.conf.js:19]: There is still Screener here in the env variable

File: visual-e2e/examples/test/configs/wdio.screener.saucelabs.conf.js [hunk 3]
```diff
@@ -1,7 +1,5 @@
 const {config} = require('./wdio.shared.conf');
```
Comment (wswebcreation) [inline visual-e2e/examples/test/configs/wdio.screener.saucelabs.conf.js:1]: This file still has screener in his name

**Review by wswebcreation — CHANGES_REQUESTED** [review]

File: visual-e2e/examples/test/configs/wdio.screener.saucelabs.conf.js (hunk 2 above)
Comment (nadvolod) [reply visual-e2e/examples/test/configs/wdio.screener.saucelabs.conf.js:19]: @wswebcreation this is what the documentation recommends

**Comment by nadvolod** [comment]
@wswebcreation sorry I missed those somehow, crazy day. I think I got them all. Also, API key is what the docs recommend so I left it. Merge if it's good please

File: visual-e2e/examples/package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test.screener.sauce": "npx wdio test/configs/wdio.screener.saucelabs.conf.js"
+    "test.screener.sauce": "npx wdio test/configs/wdio.visual.saucelabs.conf.js"
```
Comment (wswebcreation) [inline visual-e2e/examples/package.json:7]:
```suggestion
    "test.visual.sauce": "npx wdio test/configs/wdio.visual.saucelabs.conf.js"
```

**Review by wswebcreation — APPROVED** [review]
Still didn't catch them all 😉 , I'll change it and merge it my friend

**Pull request merged**